.azure-pipelines/generation-templates/python-beta.yml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  - name: repoName
+    type: string
+
+steps:
+- pwsh: '$(scriptsDirectory)/clean-python-files.ps1'
+  displayName: 'Remove generated models and requests from the repo generated folder'
+  env:
+    RepoModelsDir: '$(Build.SourcesDirectory)/${{ parameters.repoName }}/msgraph_beta/generated'
```

Comment:
my understanding is that we duplicated the file just for the msgraph_beta segment here (x2), could we not instead add an additional parameter to original template?